core/src/orderbook/streamed/updating_orderbook.rs

```diff
@@ -194,7 +196,7 @@ impl UpdatingOrderbook {
                     .block_timestamp(meta.block_hash)
                     .await?;
                 context.orderbook.handle_event_data(
-                    data,
+                    EventStatus::Added(data),
```

Comment:
I suspect that we don't need match on `EventStatus::{Added, Removed}` anymore. Maybe we can remove it?

Reply:
The way we use the past events, yes we could remove this. On the other I hand since we already have this code it might make sense to keep this flexibility.

Reply:
Personally, I think the price of maintaining dead code is usually not worth it. We can always roll back these changes if we need it again (since the original implementation with `EventData::{Added, Removed}` will be in version control.